components/Banner/Banner.tsx

```diff
@@ -18,7 +20,7 @@ const OuterWrapper = styled.div`
 
 const InnerWrapper = styled.div`
   max-width: var(--desktop-max-width);
-  height: 125px;
+  height: var(--banner-height);
```

Comment:
Use a global var so that this value can be used in calculations elsewhere (we'll also need this for the mobile views soon)